nexus/lib/pwscf_input.py

```diff
@@ -204,6 +204,7 @@ class PwscfInputBase(DevBase):
         'esm_efield','fcp_mu','london_c6','london_rvdw','xdm_a1','xdm_a2',
         # 6.3 additions
         'block_1','block_2','block_height','zgate','ts_vdw_econv_thr',
+        'starting_charge',
```

Comment:
Please place "starting_charge" back in the "real_arrays" list.  This reverts a fix made in PR #1656.

Reply:
Do you mean like this?
   real_arrays = [
        'celldm', 'starting_magnetization', 'hubbard_alpha', 'hubbard_u',
        'hubbard_j0', 'hubbard_beta', 'hubbard_j',
        'starting_ns_eigenvalue', 'angle1', 'angle2', 'fixed_magnetization',
        'fe_step', 'efield_cart', 'london_c6', 'london_rvdw',
        starting_charge,
         ]

Reply:
Yes, except it needs to be a string, i.e. starting_charge -> 'starting_charge'.

Reply:
starting_charge still needs to be removed from the "floats" list